src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/DiscretePhotonEnergy/Builder/src/energy.hpp

```diff
@@ -0,0 +1,9 @@
+Builder& energy( double energy ){
```

Comment:
I'll only do this here: add one for Quantity<MeV> and have the double energy function call this one.
This is a simple one to do.